pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -271,11 +281,15 @@ func (m *Manager) processShutdownEvent() error {
 	case <-c:
 		break
 	case <-time.After(m.shutdownGracePeriodRequested):
-		klog.V(1).InfoS("Shutdown manager pod killing time out", "gracePeriod", m.shutdownGracePeriodRequested)
+		msg := "Shutdown manager pod killing time out"
+		m.recorder.Event(m.nodeRef, v1.EventTypeWarning, kubeletevents.NodeShutdown, msg)
+		klog.V(1).InfoS(msg, "gracePeriod", m.shutdownGracePeriodRequested)
 	}
 
 	m.dbusCon.ReleaseInhibitLock(m.inhibitLock)
-	klog.V(1).InfoS("Shutdown manager completed processing shutdown event, node will shutdown shortly")
+	msg := "Shutdown manager completed processing shutdown event, node will shutdown shortly"
+	m.recorder.Event(m.nodeRef, v1.EventTypeNormal, kubeletevents.NodeShutdown, msg)
+	klog.V(1).InfoS(msg)
```

Comment:
Should we record the node? It would also probably be better to put the active pods in a label pair as well, if we can.

Reply:
the node gets recorded via the nodeRef...

Reply:
The active pods would be a bit verbose for the Event log. I vote leave them out.